core/domain/role_services.py

```diff
@@ -287,7 +293,9 @@ def get_role_actions():
     return copy.deepcopy(_ROLE_ACTIONS)
 
 
-def log_role_query(user_id, intent, role=None, username=None):
+def log_role_query(
+        user_id: str, intent: str, role: Optional[str]=None,
+        username: Optional[str]=None) -> None:
```

Comment:
```suggestion
def log_role_query(
    user_id: str, 
    intent: str, 
    role: Optional[str] = None,
    username: Optional[str] = None
) -> None:
```

Reply:
Not done

Reply:
Sorry @vojtechjelinek I have made the requested change in the new commit

Reply:
still not done, you're missing the spaces around "="

Reply:
Made the changes now @vojtechjelinek